src/App.js

```diff
@@ -23,13 +27,25 @@ function App() {
       //Reset states on new zipcode
       setHoverEvent(null);
       setLocFilt(null);
+      setCardIndex(0);
 
     }
   }, [currZip]);
 
+  //Card index utilizes the hoverEvent to highlight the card's respective marker
+  useEffect(() => {
+    if(isMobile && events != null){
+      setHoverEvent((('location' in events[cardIndex] && 'location' in events[cardIndex]['location'] && 'latitude' in events[cardIndex]['location']['location']) ? "" + events[cardIndex]['location']['location']['latitude'] + "&" + events[cardIndex]['location']['location']['longitude'] : null));
+    }
+  }, [cardIndex]);
+
+
   return (
     <div className="app">
       <SearchBar currZip={currZip} updateZip={(newZip) => setCurrZip(newZip)} events={events} updatedHover={(newHover) => setHoverEvent(newHover)} locFilt={locFilt}/>
+      {events !== null && isMobile &&
+        <MobileList events={events} updatedHover={(newHover) => setHoverEvent(newHover)} locFilt={locFilt} cardIndex={cardIndex} updateCardIndex={(update) => setCardIndex(update)}/>
```

Comment:
Are the lambdas serving a purpose here? (i.e. making sure only a single arg gets passed in?) Or could this more simply be ‘updatedHover=setHoverEvent’